polyglot/piranha/test-resources/go/feature_flag/system_1/const_same_file/configurations/rules.toml

```diff
@@ -20,19 +20,19 @@ query = """
             field: (field_identifier) @func_id
         )
         arguments: (argument_list
-            (interpreted_string_literal) @arg_str_literal
+            (identifier) @arg_id
         )
     )
     (#eq? @func_id "BoolValue")
-    (#eq? @arg_str_literal "\\\"@stale_flag_name\\\"")
+    (#eq? @arg_id "@const_id")
 ) @call_exp
 """
 replace = "false"
 replace_node = "call_exp"
-holes = ["stale_flag_name"]
+holes = ["const_id"]
 
 [[rules]]
-name = "delete_const_str_literal"
+name = "find_const_str_literal"
```

Comment:
let's reorder the rules? Because I assume the flow is `find_const_str_literal` -> `false_flag`

Reply:
Good point.